traversal/procedure/ProcedureVertex.java

```diff
@@ -63,7 +63,6 @@
         > extends TraversalVertex<ProcedureEdge<?, ?>, PROPERTIES> {
```

Comment:
Removing crutches that were needed during transition from edge-ordered to vertex-ordered plan.